src/DatePickerInput.js

```diff
@@ -96,14 +96,8 @@ const DatePickerInput = React.createClass({
     }
   },
 
-  stopPropagation(e) {
-    if (this.props.closeOnClickOutside) {
-      e.stopPropagation();
-    }
-  },
-
-  hide() {
-    if (this.state.showing) {
+  hide(e) {
+    if (e.target !== this.refs.datePickerInput.getDOMNode() && this.state.showing) {
```

Comment:
if the prop `floating` is `false` for some reasons `e` is undefined -> `e.target` causes a crash